docs/resources/Sticker.md

```diff
@@ -0,0 +1,101 @@
+| GUILD    | 2     | a sticker uploaded to a Boosted guild for the guild's members                 |
+
+###### Sticker Format Types
+
+| Type   | Value |
+| ------ | ----- |
+| PNG    | 1     |
+| APNG   | 2     |
+| LOTTIE | 3     |
+
+###### Example Sticker
+
+```json
+{
+  "id": "749054660769218631",
+  "name": "Wave",
+  "tags": "wumpus, hello, sup, hi, oi, heyo, heya, yo, greetings, greet, welcome, wave, :wave, :hello, :hi, :hey, hey, \ud83d\udc4b, \ud83d\udc4b\ud83c\udffb, \ud83d\udc4b\ud83c\udffc, \ud83d\udc4b\ud83c\udffd, \ud83d\udc4b\ud83c\udffe, \ud83d\udc4b\ud83c\udfff, goodbye, bye, see ya, later, laterz, cya",
+  "type": 1,
+  "format_type": 3,
+  "description": "Wumpus waves hello",
+  "asset": "",
+  "pack_id": "847199849233514549",
+  "sort_value": 12
+}
+```
+
+### Sticker Pack Object
+
+Represents a pack of standard stickers.
+
+###### Sticker Structure
+
+| Field            | Type                                                               | Description                                                               |
+| ---------------- | ------------------------------------------------------------------ | ------------------------------------------------------------------------- |
+| id               | snowflake                                                          | id of the sticker pack                                                    |
+| stickers         | array of [sticker](#DOCS_RESOURCES_STICKER/sticker-object) objects | the stickers in the pack                                                  |
+| name             | string                                                             | name of the sticker pack                                                  |
+| sku_id           | snowflake                                                          | id of the pack's SKU                                                      |
+| cover_sticker_id | snowflake                                                          | id of a sticker in the pack which is shown as the pack's icon             |
+| description      | string                                                             | description of the sticker pack                                           |
+| banner_asset_id  | snowflake                                                          | id of the sticker pack's [banner image](#DOCS_REFERENCE/image-formatting) |
+
+###### Example Sticker Pack
+
+```json
+{
+  "id": "847199849233514549",
+  "stickers": [],
+  "name": "Wumpus Beyond",
+  "sku_id": "847199849233514547",
+  "cover_sticker_id": "749053689419006003",
+  "description": "Say hello to Wumpus!",
+  "banner_asset_id": "761773777976819732"
+}
+```
+
+## List Nitro Sticker Packs % GET /sticker-packs
+
+Returns the list of sticker packs available to Nitro subscribers.
+
+###### Response Structure
+
+| Field         | Type                                                                         |
```

Comment:
A description should be here too maybe

Reply:
I didn't put a desc here because the text above the table pretty much explains it